src/libraries/System.Runtime.Numerics/src/System/Numerics/BigIntegerCalculator.DivRem.cs

```diff
@@ -68,9 +72,16 @@ public static void Divide(ReadOnlySpan<uint> left, ReadOnlySpan<uint> right, Spa
             Debug.Assert(left.Length >= right.Length);
             Debug.Assert(quotient.Length == left.Length - right.Length + 1);
             Debug.Assert(remainder.Length == left.Length);
+            DummyForDebug(quotient);
+            DummyForDebug(remainder);
 
-            left.CopyTo(remainder);
-            Divide(remainder, right, quotient);
+            if (right.Length < DivideBurnikelZieglerThreshold || left.Length - right.Length < DivideBurnikelZieglerThreshold)
+            {
+                left.CopyTo(remainder);
+                DivideGrammarSchool(remainder, right, quotient);
+            }
+            else
```

Comment:
nit: code style requirements (https://github.com/dotnet/runtime/blob/main/docs/coding-guidelines/coding-style.md) require braces for the `else`:
> Using braces is always accepted, and required if any block of an if/else if/.../else compound statement uses braces or if a single statement body spans multiple lines.

My own personal preference for code in the numerics area is then to never omit braces as it helps avoid bugs and improve readability.